docs/providers/mycat/index.rst

```diff
@@ -0,0 +1,32 @@
+MyCat
+=====
+
+This database provider allows Entity Framework Core to be used with `MyCat <https://github.com/MyCATApache/Mycat-Server>`_. The provider is maintained as part of the `Pomelo Foundation Project <https://github.com/PomeloFoundation/Entity-Framework-Core-MyCat-Proxy>`_.
+
+.. contents:: `In this article:`
+    :depth: 2
+    :local:
+
+Install
+-------
+
+Download and run `the latest release from the project site <https://github.com/PomeloFoundation/Entity-Framework-Core-MyCat-Proxy/releases>`_.
+
+Get Started
+-----------
+
+The following resources will help you get started with this provider.
+ * `Getting started documentation <https://github.com/aspnet/EntityFramework.Docs/issues/252>`_
+ * `Getting started video <https://www.youtube.com/watch?v=q0CXfFNtMZo>`_
+
+Supported Database Engines
+--------------------------
+
+  * MyCat
```

Comment:
Any particular version restrictions or anything I should list here? For example, on the SQL Server page we list that we work with SQL Server 2008 onwards.